src/components/features/AllowedCallsEncoder/AllowedCallsEncoder.tsx

```diff
@@ -182,6 +195,43 @@ const AllowedCallsEncoder: React.FC = () => {
     setAllowedCallTypes(updatedCallTypes);
   };
 
+  useEffect(() => {
+    // TODO get address abi if exists
+    async function getAbi() {
+      if (!isAddress(allowedAddressValue)) {
+        setAllowedAddressIsContract(false);
+        setAllowedAddressName(undefined);
+        setAllowedAddressAbi([]);
+        return;
+      }
+
+      const { address } = await request(
+        'https://explorer.execution.mainnet.lukso.network/api/v1/graphql',
```

Comment:
Suggest getting the explorer URL using the hook `useContext(NetworkContext)` and accessing the `rpcUrl` property.

<img width="500" height="37" alt="image" src="https://github.com/user-attachments/assets/23599708-3aa2-4d00-a060-a58e6b182fd2" />